### PR TITLE
Cinema spring 02

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: src/main/java/com/dev/cinema/controllers/MovieController.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    private final MovieService movieService;
+    private final MovieMapper movieMapper;
+
+    @Autowired
+    public MovieController(MovieService movieService, MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
+
+    @PostMapping("/create")
```
Comment (dariiabutorina) [inline src/main/java/com/dev/cinema/controllers/MovieController.java]: Let's not use CRUD naming in the URL

File: src/main/java/com/dev/cinema/controllers/MovieSessionMapper.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        this.movieService = movieService;
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    protected MovieSession toMovieSessionModel(MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession movieSession = new MovieSession();
+        CinemaHall cinemaHall = cinemaHallService.get(movieSessionRequestDto.getCinemaHallId());
+        Movie movie = movieService.get(movieSessionRequestDto.getMovieId());
+
+        movieSession.setShowTime(LocalDateTime.parse(movieSessionRequestDto.getShowTime(),
```
Comment (dariiabutorina) [inline src/main/java/com/dev/cinema/controllers/MovieSessionMapper.java]:
```suggestion
        Movie movie = movieService.get(movieSessionRequestDto.getMovieId());
        movieSession.setShowTime(LocalDateTime.parse(movieSessionRequestDto.getShowTime(),
```

**Review by dariiabutorina — CHANGES_REQUESTED** [review]
Merge the previous PR, please, so the old changes won't appear in here

**Review by RomanDubovskyi — COMMENTED** [review]
you have here files not related to this PR (previous HW) let's remove them
\+ change PR name it's uninformative

**Pull request closed without merging**